### PR TITLE
fix: add tags to IAM Policy

### PR DESCRIPTION
**Pull request by rajgandhi9**
Issue: Add tags to IAM Policy

Description:
Adding the user provided tags to the IAM policy created by this terraform module

Additional Info:
N/A

### CODE REVIEW

**Comment by afiune** [comment]
Make it so! 🙌🏽

**Comment by afiune** [comment]
@rajgandhi9 Thank you so much for your contribution! 

In order to accept this PR, we need you to sign your commits for security reasons. This is part of our development and contributing guidelines: https://github.com/lacework/terraform-aws-config/blob/main/DEVELOPER_GUIDELINES.md#signing-commits

**Comment by rajgandhi9** [comment]
@afiune - Signed the commit. Thanks.

File: main.tf [hunk 1]
```diff
@@ -344,3 +345,4 @@ resource "lacework_integration_aws_ct" "default" {
 
   depends_on = [time_sleep.wait_time]
 }
+
```
Comment (afiune) [inline main.tf:348]:
```suggestion
```

**Comment by afiune** [comment]
make it so

**Comment by rajgandhi9** [comment]
@afiune - Anything else required from my side? Thanks.

**Comment by afiune** [comment]
make it so

**Review by afiune — APPROVED** [review]
tenor-217425937

**Comment by rajgandhi9** [comment]
@afiune - Thank you. Will you be releasing a new version with these changes?

**Comment by afiune** [comment]
@rajgandhi9 Just released it! 💯  Thank you again for your contribution! 🙌🏽

**Pull request merged**